cleanlab/datalab/issue_manager/noniid.py

```diff
@@ -164,6 +164,16 @@ def find_issues(self, features: Optional[npt.NDArray] = None, **kwargs) -> None:
             },
         )
 
+        num_issues = self.issues[f"is_{self.issue_name}_issue"].sum()
+        if self.p_value >= 0.05:
+            # Set all issues to False if the p-value is greater than 0.05
```

Comment:
```suggestion
        SIGNIFICANCE_THRESHOLD = 0.05
        if self.p_value >= SIGNIFICANCE_THRESHOLD:
            # Set all issues to False if the p-value is insignificant
```

Reply:
could alternatively make `SIGNIFICANCE_THRESHOLD` an optional hyperparameter of this issuemanager

Reply:
Added a `significance_threshold` parameter to the constructor.

We should remember not to confuse this with a `threshold` typically used to flag individual data points as having issues.